src/API/WesternStatesWater.WestDaat.Client.Functions/System.cs

```diff
@@ -10,135 +9,60 @@
-
-            // This cannot run thru CreateOKResponse because the serializer options will convert enum to string
+            var results = await systemManager.LoadFilters();
+            // This cannot run through CreateOKResponse because the serializer options will convert enum to string
             // This will break the UI.
             var data = req.CreateResponse(HttpStatusCode.OK);
             await data.WriteAsJsonAsync<object>(results, new JsonObjectSerializer(
-            new JsonSerializerOptions
-            {
-                PropertyNamingPolicy = JsonNamingPolicy.CamelCase
-            }));
+                new JsonSerializerOptions
+                {
+                    PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+                }));
             return data;
         }
 
-        [Function(nameof(GetOwnerClassifications))]
-        public async Task<HttpResponseData> GetOwnerClassifications([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/ownerclassifications")] HttpRequestData request)
-        {
-            _logger.LogInformation("Getting Owner Classifications");
-
-            var results = await _systemManager.GetAvailableOwnerClassificationNormalizedNames();
-
-            return await CreateOkResponse(request, results);
-        }
-
-        [Function(nameof(GetWaterSourceTypes))]
-        public async Task<HttpResponseData> GetWaterSourceTypes([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/watersourcetypes")] HttpRequestData request)
-        {
-            _logger.LogInformation("Getting Water Source Types");
-
-            var results = await _systemManager.GetAvailableWaterSourceTypeNormalizedNames();
-
-            return await CreateOkResponse(request, results);
-        }
-
-        [Function(nameof(GetStates))]
-        public async Task<HttpResponseData> GetStates([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/states")] HttpRequestData request)
-        {
-            _logger.LogInformation("Getting States");
-
-            var results = await _systemManager.GetAvailableStateNormalizedNames();
-
-            return await CreateOkResponse(request, results);
-        }
-
-        [Function(nameof(GetRiverBasinNames))]
-        public async Task<HttpResponseData> GetRiverBasinNames([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/RiverBasins")] HttpRequestData request)
-        {
-            var result = _systemManager.GetRiverBasinNames();
-
-            return await CreateOkResponse(request, result);
-        }
-
         [Function(nameof(GetRiverBasinPolygonsByName))]
-        public async Task<HttpResponseData> GetRiverBasinPolygonsByName([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "system/RiverBasins")] HttpRequestData request)
+        public async Task<HttpResponseData> GetRiverBasinPolygonsByName(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "system/RiverBasins")]
```

Comment:
not really related to the pr but can we camel case this to `system/riverBasins`?